tests/unit/aofrw.tcl

```diff
@@ -179,26 +179,25 @@ start_server {tags {"aofrw external:skip"} overrides {aof-use-rdb-preamble no}}
     }
 
     test {BGREWRITEAOF is delayed if BGSAVE is in progress} {
-        r multi
+        r config set rdb-key-save-delay 1000000
```

Comment:
are there any keys? maybe we should not rely on past tests, and instead add FLUSHALL and add a single key

Reply:
I test there is a key left.